src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.js

```diff
@@ -36,6 +39,23 @@ function extractAttachmentsFromReport(report, reportActions) {
         if (!ReportActionsUtils.shouldReportActionBeVisible(action, key)) {
             return;
         }
+
+        // We're handling receipts differently here because receipt images are not
+        // part of the report action message, the images are constructed client-side
+        if (ReportActionsUtils.isMoneyRequestAction(action)) {
+            const transaction = TransactionUtils.getTransaction(action.originalMessage.IOUTransactionID);
```

Comment:
I think that not every IOU action has the `originalMessage.IOUTransactionID`, could we access it safely and return early if empty?

Reply:
+1 pay actions don't have the `transactionID`. Let's fix it in this PR to avoid any crashes.